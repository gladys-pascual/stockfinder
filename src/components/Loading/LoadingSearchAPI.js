import SyncLoader from "react-spinners/SyncLoader";
import "./Loading.scss";

const LoadingSearchAPI = () => {
  return (
    <div className="loading-search-api">
      <SyncLoader color="#7bcbc4" size="15px" margin="2px" />
    </div>
  );
};

export default LoadingSearchAPI;
