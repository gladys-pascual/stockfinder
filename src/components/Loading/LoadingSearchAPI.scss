.loading-search-api {
  padding-top: 30px;
}
