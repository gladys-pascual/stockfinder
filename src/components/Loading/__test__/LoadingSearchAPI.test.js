import { render } from "@testing-library/react";
import LoadingSearchAPI from "../LoadingSearchAPI";

test("renders LoadingSearchAPI component as expected", () => {
  const { container } = render(<LoadingSearchAPI />);
  expect(container).toMatchSnapshot();
});
