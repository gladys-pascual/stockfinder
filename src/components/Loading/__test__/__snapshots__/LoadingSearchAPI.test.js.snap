// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders LoadingSearchAPI component as expected 1`] = `
<div>
  <div
    class="loading-search-api"
  >
    <span
      class="css-0"
    >
      <span
        class="css-158729u"
      />
      <span
        class="css-jyyt77"
      />
      <span
        class="css-uj62sn"
      />
    </span>
  </div>
</div>
`;
